core/tests/protractor_desktop/explorationStatisticsTab.js

```diff
@@ -130,6 +130,12 @@ describe('Issues visualization', function() {
       function(elem) {
         elem.setValue(1.0);
       });
+    adminPage.editConfigProperty(
+      'Exposes the Improvements Tab for creators in the exploration editor.',
+      'Boolean',
+      function(elem) {
+        elem.setValue(false);
```

Comment:
Er, why false? I thought we _want_ to show the improvements tab...?

Reply:
There should be a TODO + issue to remove this in the future, then? (Maybe after the tab is working smoothly in prod for 1-2 months?)

Reply:
Done.